examples/routing_service/routing_service_file_adapter/cpp/.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode/
+build/
+*.o
```

Comment:
Should we maybe add these patterns to our global `.gitignore` instead of creating a new one in this specific directory?

Reply:
Done.